CONTRIBUTING.md

```diff
@@ -88,16 +88,7 @@ yarn develop
 
 The server (API) is available at http://localhost:1337
 
-**WARNING** ⚠️ If you've followed the recommended setup, you should not be able to reach the administration panel at http://localhost:1337/admin.
-
-**Start the administration panel server**
```

Comment:
We also need this part if you want to be able to modify the front-end files of the packages.

Reply:
Ah! So you have to run `yarn develop` in both places?

Reply:
Yes, we do when we need to modify the core packages